frame.go

```diff
@@ -1786,6 +1807,14 @@ func (f *framer) writeStringList(l []string) {
 	}
 }
 
+func (f *framer) writeUnset() {
+	// Protocol version 4 specifies that bind variables do not require having a
+	// value when executing a statement.   Bind variables without a value are
+	// called 'unset'. The 'unset' bind variable is serialized as the int
+	// value '-2' without following bytes.
+	f.writeInt(-2)
```

Comment:
Other drivers throw an exception here.  I opt'd to let the user hang themselves which potentially is not the right answer.  By inserting -1 we insert nulls which has the side effect of causing additional tombstones in cassandra.

Would we prefer i panic if protocol is < 4?